base/dcp_client.go

```diff
@@ -329,15 +336,38 @@ func (dc *DCPClient) rollback(vbID uint16) (err error) {
 	return nil
 }
 
+func (dc *DCPClient) getOpenStreamOptions() (*gocbcore.OpenStreamOptions, error) {
```

Comment:
I don't think we expect to change the set of collection IDs for a DCP client post-initialization.  Given that, can we just accept the set of collection IDs as an input parameter?  Or at least, compute them in NewDCPClient, instead of adding the bucket reference to DCPClient?  That would let us avoid problems like the bucket getting invalidated underneath the client.

Reply:
Moved to collections API.